image_generation/stable_diffusion_1_5/cpp/README.md

```diff
@@ -1,51 +1,57 @@
-# OpenVINO Stable Diffusion (with LoRA) C++ pipeline
-The pure C++ text-to-image pipeline, driven by the OpenVINO native API for Stable Diffusion v1.5 with LMS Discrete Scheduler, supports both static and dynamic model inference. It includes advanced features like Lora integration with safetensors and OpenVINO extension for tokenizer. This demo has been tested on the Windows platform.
+# OpenVINO Stable Diffuison (with LoRA) C++ pipeline
+The pure C++ text-to-image pipeline, driven by the OpenVINO native API for Stable Diffusion v1.5 with LMS Discrete Scheduler, supports both static and dynamic model inference. It includes advanced features like LoRA integration with safetensors and [OpenVINO extension for tokenizers](https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/custom_operations/user_ie_extensions/tokenizer/python/README.md). This demo has been tested on Windows and Linux platform.
 
-## Step 1: Prepare environment
+> [!NOTE]
+>This tutorial assumes that you are located in `<openvino.genai repo>/image_generation/stable_diffusion_1_5/cpp/` and all paths are relative to this folder.
+
+## Step 1: Prepare build environment
 
 C++ Packages:
-* CMake: Cross-platform build tool
-* OpenVINO: Model inference
+* [CMake](https://cmake.org/download/): Cross-platform build tool
+* [OpenVINO](https://docs.openvino.ai/2023.2/openvino_docs_install_guides_overview.html): Model inference
 * Eigen3: Lora enabling
 
-SD Preparation can be auto implemented with the `build_dependencies.sh` script. 
-It provides 2 ways to install `OpenVINO 2023.1.0`: 
-[conda-forge](https://anaconda.org/conda-forge/openvino) and 
-[Download archives](https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.1/windows/).
-
+SD preparation could be auto implemented with `scripts/build_dependencies.sh`. This script provides 2 ways to install `OpenVINO 2023.1.0`: [conda-forge](https://anaconda.org/conda-forge/openvino) and [Download archives](https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.1/windows/).
 ```shell
-cd scripts
-chmod +x build_dependencies.sh
-./build_dependencies.sh
+chmod +x ./scripts/build_dependencies.sh
```

Comment:
It was there before, so you can view it as out of this PR scope. Can you push `build_dependencies.sh` version with chmod +x already applied and remove this line

Reply:
fixed